### DISCUSSION TITLE
Parkfunktion

### DISCUSSION THREAD

**Discussion by marc950** [Stephan-S/FS19_AutoDrive#2040] [Q&A]
Hallo 

meine Parkfunktion ist defekt: keine P Zuweisung habe den Mauszeiger über dem P drücke die rechte Maustaste keine Reaktion

**Comment by ChrisBS91** [comment]
HI 
Du musst dabei die "rechte-Strg" Taste gedrückt halten und dann mit Rechtsklick auf das P
